lib/mixins/bindings.js

```diff
@@ -0,0 +1,46 @@
+const { getMapValueByKey } = require('../utils');
```

Comment:
I created special folder only with mixins to avoid circular ref between js modules.